lib/gql/gql.go

```diff
@@ -0,0 +1,28 @@
+package gql
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+)
+
+func AsGQL(ctx context.Context, req interface{}) (*string, error) {
+	// Assume object is a flat keystruct
```

Comment:
I don't think this can always be assumed. For a lot of the queries I think there will be nested objects. Unless maybe I'm misunderstanding what this means

Reply:
Was gonna come back in a separate PR and extend it but maybe I just chuck the interface recursion here

Gimme 30m

Reply:
Done